store/postgres/src/store.rs

```diff
@@ -1619,9 +1615,6 @@ impl EthereumCallCache for Store {
             {
                 use crate::db_schema::eth_call_cache::dsl;
 
-                // Once we stop getting these logs, this code can be removed.
-                trace!(self.logger, "Updating eth call cache entry");
```

Comment:
We should only remove this log if we remove the call cache migration (this `else` branch). Which I think we're ready to do, I fund only very few of these in the recent hosted service logs.

Reply:
I don't understand why this log is useful - what does it tell us?

Reply:
It tell us that there is some entry in the old format that's being updated to the new one. As long as this is happening for some entries, we want to keep this fallback.

Reply:
Ok. I've put that statement back in and rebased the PR against latest master